CHANGES.rst

```diff
@@ -11,6 +11,11 @@
 
 - 100% branch coverage.
 
+- Make the C and Python `TimeStamp` objects behave more alike. The
+  Python version now produces the same `repr` and `.raw()` output as
+  the C version, and has the same hashcode. In addition, the Python
+  version is now supports ordering and equality like the C version.
+
```

Comment:
Ahh. It's just a bullet point, not a section, and the rest of the file has single lines between bullet points just like this. (It is somewhat inconsistent on the use of single vs double backticks, though, I went with what seemed newest.)